public/v4/apps/tubesync.yml

```diff
@@ -0,0 +1,41 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: ghcr.io/meeb/tubesync:$$cap_tubesync_version
+        volumes:
+            - $$cap_appname-config:/config
+            - $$cap_appname-downloads:/downloads
+        restart: always
+        environment:
+            PUID: 1000
+            PGID: 1000
+            TZ: America/Chicago
+            HTTP_USER: $$cap_username
+            HTTP_PASS: $$cap_password
+        caproverExtra:
+            containerHttpPort: '4848'
+caproverOneClickApp:
+    variables:
+        - id: $$cap_tubesync_version
+          label: TubeSync Version
+          defaultValue: 'v0.12.0'
+          description: Check out their docker page for the valid tags https://github.com/meeb/tubesync/tags
+          validRegex: /^([^\s^\/])+$/
+        - id: $$cap_username
+          label: TubeSync User
+          defaultValue: ''
+          description: 'Username for TubeSync'
+        - id: $$cap_password
+          label: Password
+          defaultValue: ''
```

Comment:
use a default password for the ease of installation, like this:
```
$$cap_gen_random_hex(16)
```

Reply:
Done.